src/newSeed/newSeed.ts

```diff
@@ -162,7 +166,9 @@ export class NewSeed {
   }
 
   setStage(route: string): void {
-    this.router.navigate(route);
+    if (!this.seedCreated) {
+      this.router.navigate(route);
```

Comment:
how are we supposed to get to stage7?

Reply:
Note this isn't just about navigation.  What about the browser back and forward buttons?

I think you simply need to:

1) clear the seedConfig and wizardData after the Seed creation _transaction_ is created (BTW, note that the Seed itself will not yet have been created at any point in this process).  Do this upon entry to stage 7.
2) hide the menu on stage7.